src/browserWindow.ts

```diff
@@ -0,0 +1,179 @@
+import {
+    window,
+    ViewColumn,
+    WebviewPanel
+} from 'vscode';
+import { D2P } from './docToPreviewGenerator';
+
+const preText = String.raw`
```

Comment:
can we define this in a `.html` likewise for the css and js?

Reply:
I changed this to load an HTML file, but breaking up into html/css/js file causes security issues, which is solved by using asWebViewUri to get an acceptable Uri to the css/js files, but then has to be inserted into the html text, which gets us back to the original problem of having the html defined as strings in the code.